fabric-input-events-v0/src/main/java/net/fabricmc/fabric/api/client/FabricMouse.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client;
+
+import net.fabricmc.fabric.impl.client.FabricMouseImpl;
+
+public interface FabricMouse {
```

Comment:
If we only expose static methods for the API, this should become a final class + private constructor.

I would also rename the class to something more akin to the purpose? FabricMouse just seems a bit weird sounding imo

Reply:
>I would also rename the class to something more akin to the purpose?

I originally named it `Mouse` but it caused a conflict with the Minecraft class when I tried to use it.

Reply:
> final class + private constructor
This seems so obvious in hindsight. I guess in my head I intended for there to be multiple implementations.